src/cosl/interfaces/cluster.py

```diff
@@ -479,6 +508,13 @@ def _get_data_from_coordinator(self) -> Optional[ClusterProviderAppData]:
 
         return data
 
+    def get_worker_config_version(self) -> Optional[str]:
+        """Fetch the worker config workload version from the coordinator databag."""
```

Comment:
```suggestion
        """Fetch the worker config version from the coordinator databag."""
```

Or, if you go with the suggestion above:
```suggestion
    def is_config_valid_for_version(self) -> bool:
        """Check whether the config we received is valid for the version we requested."""
                data = self._get_data_from_coordinator()
        if data:
            return data.workload_version_supported
        return False
        ```